File: Java/src/Car.java
public class Car {
    Integer id;
    String license;
    Account driver;
    private Integer passenger;

    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;
        this.passenger = 3;
    }

    //METODO
     void printDataCar() {
        System.out.println("License: " + license + " Driver: " + driver.document + " Pasajeros:  " + passenger);

    }

    void cambiarplaca(String license) {
        this.license = license;
    }

    void cmabiopa(Integer passenger) {
        if (passenger<=4){
            System.out.println("El numero de pasajeros es correcto");
            this.passenger=passenger;
        }else if(passenger>3){
            System.out.println("El numero de pasajeros es incorrecto");
        }
    }

    public Integer getPassenger() {
        return passenger;
    }

}

File: PHP/car.php
<?php
require_once 'Account.php';

class car
{
    public $id;
    public $license;
    public $driver;
    public $passenger;

    public function __construct($license, $driver)
    {
        $this->license = $license;
        $this->driver = $driver;
    }

    public function printDataCar()
    {
        echo "
        Licencia: $this->license 
        Driver: {$this->driver->name} 
        Número de pasajeros: $this->passenger
        
    ";
    }

    public function validarpasajeros($passanger)
    {
        if($passanger>=4){
            echo"El numero de pasajeros es correcto";
            $this->passenger=$passanger;
        }else{
            echo"El numero de pasajeros es incorrecto";
        }
    }
}

?>